pos_pricelist/static/src/js/models.js

```diff
@@ -410,18 +416,20 @@ function pos_pricelist_models(instance, module) {
             var items = [], i, len;
             for (i = 0, len = db.pricelist_item_sorted.length; i < len; i++) {
                 var item = db.pricelist_item_sorted[i];
-                if ((item.product_id === false
-                    || item.product_id[0] === product.id) &&
-                    (item.categ_id === false
-                    || categ_ids.indexOf(item.categ_id[0]) !== -1) &&
-                    (item.price_version_id[0] === version.id)) {
+                if (
+                    (item.product_tmpl_id === false || item.product_tmpl_id[0] === product.id)
```

Comment:
I believe it must be  like this:


      (item.product_tmpl_id === false || item.product_tmpl_id[0] === product.product_tmpl_id)